src/System Application/App/AI/src/Azure OpenAI/AzureOpenAIImpl.Codeunit.al

```diff
@@ -459,6 +471,12 @@ codeunit 7772 "Azure OpenAI Impl" implements "AI Service Name"
         EmptySecretText: SecretText;
     begin
         ClearLastError();
+
+        if not IsBillingTypeAuthorized(AOAIAuthorization, CallerModuleInfo) then begin
+            Error := StrSubstNo(BillingTypeAuthorizationErr, CopilotCapabilityImpl.GetCapabilityName(), CopilotCapabilityImpl.GetCopilotBillingType());
+            Error(Error);
```

Comment:
No reason for setting into the variable Error first. 
Just use `Error(StrSub....);`